tracer/src/Datadog.Trace/ClrProfiler/Instrumentation.cs

```diff
@@ -159,14 +165,23 @@ public static void Initialize()
 
             InitializeNoNativeParts();
             var tracer = Tracer.Instance;
-            InitializeLiveDebugger();
 
             if (tracer is null)
             {
                 Log.Debug("Skipping TraceMethods initialization because Tracer.Instance was null after InitializeNoNativeParts was invoked");
             }
             else
             {
+                try
+                {
+                    Log.Debug("Initializing async methods.");
```

Comment:
```suggestion
```

Reply:
May be I should change it to `Log.Debug("Initializing Remote Configuration management.");`, instead of removing?

Reply:
Maybe it is more useful to do that after we know whether `datadog-agent` supports it or not, and then we can either say we've initialized it successfully or that we haven't (and the reason why)? I think that'd probably be more useful for diagnostics.

Reply:
Agree, done at d248fb28c3142bad08ba7e94f8ba2412d5f8ec75